api/src/Entity/User.php

```diff
@@ -71,13 +75,13 @@
     security: "is_granted('ROLE_ADMIN') or object == user",
     provider: CurrentUserProvider::class,
 )]
-#[GetCollection(security: "is_granted('ROLE_ADMIN')")]
 #[GetCollection(
-    uriTemplate: '/customers',
+    uriTemplate: '/repairers/{repairer_id}/customers',
+    uriVariables: ['repairer_id'],
+    requirements: ['repairer_id' => '\d+'],
     openapi: new Model\Operation(
         summary: 'Retrieves customers from my repair\'s shop',
         description: 'Retrieves customers from my repair\'s shop'),
-    security: "is_granted('ROLE_ADMIN') or is_granted('ROLE_BOSS') or is_granted('ROLE_EMPLOYEE')",
```

Comment:
I think you could let security here and use a voter instead of write security in the provider?